maestro-test/src/test/kotlin/maestro/test/IntegrationTest.kt

```diff
@@ -2795,6 +2798,48 @@ class IntegrationTest {
         )
     }
 
+    @Test
```

Comment:
Can we add another test with a few commands that fails after the first one which then asserts that the `onFlowComplete` hook triggers correctly?

Reply:
ditto